src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,145 @@
+package duke;
+
+import duke.task.Task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents duke chatbot Ui that user sees.
+ */
+public class Ui {
+
+    /**
+     * Prints duke chatbot start up message that user sees when he/her opens duke chatbot.
+     */
+    public void printStartUpUi() {
+        System.out.println("========================================================================================");
+        System.out.println("Hello! I'm Duke, your personalized chatbot to arrange your tasks!");
+        System.out.println("Duke allows you to freely add, delete, mark task status, search tasks by time, " +
+                "\nand list all tasks");
+        System.out.println("________________________________________________________________________________________");
+        System.out.println("Please type 'help' command for more information on Duke task types and command format.");
+        System.out.println("========================================================================================");
+    }
+
+    /**
+     * Prints duke information message that user sees when he/her uses help command to get information about duke chatbot.
+     * The information includes task type, command format, and other notes.
+     */
+    public void printDukeInfo() {
+        System.out.println("________________________________________________________________________________________");
+        System.out.println("There are 3 types of task implemented: "
+                + "\n1. todo     : tasks without any date/time attached to it"
+                + "\n2. deadline : tasks that need to be done before a specific date/time"
+                + "\n3. event    : tasks that start at a specific time and ends at a specific time");
+        System.out.println("________________________________________________________________________________________");
```

Comment:
These lines are re-used throughout Ui.java for displaying output. Should you perhaps extract it to a static constant so the formatting can be kept consistent without copying and pasting the string?